python/sglang/srt/models/llama.py

```diff
@@ -476,14 +496,15 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
                 # Skip loading kv_scale from ckpts towards new design.
                 if name.endswith(".kv_scale") and name not in params_dict:
                     continue
-                if name in params_dict.keys():
-                    param = params_dict[name]
-                    weight_loader = getattr(
-                        param, "weight_loader", default_weight_loader
+                param = params_dict[name]
```

Comment:
nit: maybe we can keep the original logic (`if name in params_dict.keys(): ... else: ...`)